projects/core/animations/animations.ts

```diff
@@ -30,14 +30,8 @@ export const TUI_PARENT_ANIMATION = trigger('tuiParentAnimation', [
 export const TUI_PARENT_STOP = trigger('tuiParentStop', [transition(':enter', [])]);
 
 export const tuiHost = trigger('tuiHost', [
-    transition(':enter', [
-        style({overflow: 'clip'}),
```

Comment:
I thought it jumps on iOS if we remove clip on :enter

Reply:
<img width="1354" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/71ec429e-39e4-44dd-80d5-8e656efe7f7a">

I just remove `style({overflow: 'clip'}),`

Reply:
fixed